tests/test_slackclient.py

```diff
@@ -25,24 +25,28 @@ def im_created_fixture():
 def test_proxy():
     proxies = {'http': 'some-bad-proxy', 'https': 'some-bad-proxy'}
     client = SlackClient('xoxp-1234123412341234-12341234-1234', proxies=proxies)
+    server = client.server
 
-    assert client.server.proxies == proxies
+    assert server.proxies == proxies
 
-    # with pytest.raises(ConnectionError):
-    #     client.rtm_connect()
+    with pytest.raises(ProxyError):
```

Comment:
i'm not sure how this test works, but is this an intentional change of the exception type? would this break existing code that depended on a `ConnectionError`?

Reply:
Currently it doesn't appear that ConnectionError is ever raised; ProxyError is.